micro-frontends/src/next-ui/Containers/patientAlergies/PatientAlergiesControl.jsx

```diff
@@ -71,18 +86,43 @@ export function PatientAlergiesControl(props) {
       AllergenKind.FOOD
     );
 
-    const allergens = [
+    return [
       ...medicationAllergens,
       ...environmentalAllergens,
       ...foodAllergens,
     ];
-    return allergens;
   };
 
+  const allergiesAndReactionsForPatient = async () => {
+    const allergiesData = [];
+    const allergiesAndReactions = await fetchAllergiesAndReactionsForPatient(patient.uuid);
+    console.log('allergiesAndReactions', allergiesAndReactions)
+    const allergies = allergiesAndReactions.entry;
+    allergies?.forEach((allergy) => {
+      const { resource } = allergy;
+      const allergen = resource.reaction[0]?.substance.coding[0].display;
+      const severity = resource.reaction[0].severity;
+      const note = resource.note && resource.note[0].text;
+      const date = new Date(resource.recordedDate);
+      const reactions =[]
+      const provider = resource.recorder?.display;
+      resource.reaction[0]?.manifestation.forEach((reaction) => {
+        reactions.push(reaction.coding[0].display);
+      });
```

Comment:
can we use map method instead of forEach? The purpose of map method of array is same